packages/core/src/store/__tests__/level-anchor-request-store.test.ts

```diff
@@ -88,37 +91,38 @@ describe('LevelDB-backed AnchorRequestStore state store', () => {
   let streamId3: StreamID
   let genesisCommit3: GenesisCommit
 
+  // use Utils to load the genesis commit for a stream so it converts CIDs for us
+  // and we avoid any issues with one having `Symbol(Symbol.toStringTag): "CID"` and one not
+  // because the getter function isn't copied by _.cloneDeep
+  async function loadGenesisCommit(
+    ceramic: CeramicApi,
+    streamId: StreamID
+  ): Promise<GenesisCommit> {
+    const commits = await ceramic.loadStreamCommits(streamId)
+    expect(commits).toBeDefined()
+    const first = commits[0].cid
```

Comment:
if all you need is the genesis commit CID you don't need to use `loadStreamCommits` for that.  The StreamID encodes the genesis commit CID.  You can just use `streamId.cid` to get it.

Reply:
Also for future reference, `loadStreamCommits` is a very expensive call.  It not just loads the stream but then also loads each individual commit's data from IPFS.  So if you just need the CID of a specific commit, it would be better to call `loadStream` and get the CID from the log in the state object.